tests/test_utils.py

```diff
@@ -94,3 +94,44 @@ def test_invalidTimespan(self):
             getMidpointFromTimespan(timespan_none_both, allowUnbounded=True)
         with self.assertRaises(ValueError):
             getMidpointFromTimespan(timespan_none_both, allowUnbounded=False)
+
+    def test_ssObjectID_to_objID_and_objID_to_ssObjectID(self):
+        """Convert between ssObjectIDs and MPC packed designations
+        """
+        allowed_strings = [ch for ch in 'ABCDEFGHIJKMNOPQRSTUVWXYZ0123456789 -'] \
+            + ['A' * i for i in range(2, 8)] + ['Z' * i for i in range(2, 8)] \
```

Comment:
Why are single-character strings not allowed?

Reply:
They are allowed.

Reply:
Oops, silly me. The first line. 🤦